quickjs-libc.c

```diff
@@ -4023,6 +4034,7 @@ static const JSCFunctionListEntry js_os_funcs[] = {
     OS_FLAG(SIGTTOU),
     JS_CFUNC_DEF("cputime", 0, js_os_cputime ),
 #endif
+    JS_CFUNC_DEF("exePath", 0, js_os_exepath ),
```

Comment:
Maybe a getter instead of a function? Just a thought, either is fine.

Reply:
Not sure we can use one of those here because its being attached to the module and not an o next but I'll take a look!